packages/client/e2e/test-c.spec.ts

```diff
@@ -0,0 +1,343 @@
+import { test, expect, Browser, Page, Locator } from '@playwright/test';
+import { assertIsNotNull, assertIsNotUndefined } from './_utils/assert';
+import { mockSearchRooms } from './_utils/mock-http';
+import { waitForYouTubeVideoToLoad } from './_utils/wait-youtube';
+
+const AVAILABLE_USERS_LIST = [
```

Comment:
Do we want to centralize this data somewhere in the new _utils folder in an other PR ?